static/styles/input_pill.css

```diff
@@ -112,6 +123,10 @@
     }
 }
 
+.search_list_item .pill-container {
+    margin-left: 5px;
+}
```

Comment:
Can we scope this CSS inside the search area container?  It's not a generic component that can be used elsewhere, and the name makes it sound like it might be.

I think concretely this means we should move these lines inside the `zulip.css` nested block that you have below.